src/actions/authActions.js

```diff
@@ -171,6 +165,11 @@ export const loginAction = (
       const keychainLogin = await canLoginWithPkFromPin(!!biometricsSetting);
       if (pin && keychainLogin) {
         wallet = await getWalletFromPkByPin(pin);
+        // eslint-disable-next-line i18next/no-literal-string
+        Instabug.setUserAttribute('address', wallet.address);
+        if (wallet.provider._network.ensAddress) {
+          Instabug.setUserAttribute('ENS', wallet.provider._network.ensAddress);
+        }
```

Comment:
@poocart do you know if `wallet.provider._network.ensAddress` is where the ENS address stored?

Reply:
We'll revisit this when @poocart is less busy